src/main/java/seedu/address/logic/commands/schedule/FindCommonTimingCommand.java

```diff
@@ -115,45 +98,39 @@ public CommandResult execute(Model model) throws CommandException {
                 }
             }
         }
-
-        // This list will contain an even number of LocalTime objects
-        // The even indexes (i.e. 0, 2, 4, ...) will be starting times
-        // The odd indexes will be ending times
-        List<LocalTime> listOfAvailableTime = new ArrayList<>();
-        int counter = 0;
+        int toggle = 0;
```

Comment:
This toggle here assumes that everything will be free at the start. Unexpected result display will occur when the first slot is blocked.
```suggestion
        int toggle = timeSlots[0];
```
This should do the trick

Reply:
Alright, amended